src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -46,4 +46,10 @@ public Response getInfo() {
     public Response getServer() {
         return response( req -> ok(systemConfig.getDataverseServer()));
     }
+    
+    @GET
+    @Path("apiTermsOfUse")
+    public Response getTermsOfUse() {
+        return response( req -> ok(systemConfig.getApiTermsOfUse()));
```

Comment:
If you want, you could change this to `allowCors(response` (like in "version", above) to allow Cross-Origin Resource Sharing.

I'm sort of wondering what's returned if no API terms of use are set.

Reply:
Thanks, pdurbin, I'll make the changes. 
If no API terms of use are set it returns ``{"status":"OK","data":{"message":"There are no API Terms of Use for this Dataverse installation."}}``

Reply:
@solhm ah, sounds fine. In general, we don't translate API messages but someday we might want to move that message to Bundle.properties.